framework/platform/bpl/include/bpl/bpl_cfg.h

```diff
@@ -84,6 +84,13 @@ namespace bpl {
 #define BPL_GW_DB_MANAGE_MODE_LEN (127 + 1) /* Maximal length of MANAGEMENT MODE string */
 #define BPL_GW_DB_OPER_MODE_LEN (127 + 1)   /* Maximal length of OPERATING MODE string */
 
+/* Default values */
+#define DEFAULT_STOP_ON_FAILURE_ATTEMPTS 1
+#define DEFAULT_RDKB_EXTENSIONS 0
+#define DEFAULT_BAND_STEERING 0
```

Comment:
Thinking about it again - lets use the following defaults:
band steering - 1
dfs_reentry - 1
client roaming - 1
rdkb extensions - 0

Reply:
Since the defaults for linux and uci can be different I feel that the common header is the wrong place to put them.

Reply:
I dissagree - we should decide on one default set of values and if some implementation wants to override the default - well, this is why we have the DB for - simply put it in the UCI DB.

Reply:
If using a signle set of default value is an option I'd like to use it.